cmd/jaeger/config.yaml

```diff
@@ -38,7 +38,8 @@ extensions:
     #   path: ./cmd/jaeger/sampling-strategies.json
     adaptive:
       sampling_store: some_store
-      initial_sampling_probability: 0.1
+      sampling:
```

Comment:
The whole struct here is "adaptive sampling", I think repeating "sampling" is redundant, does not add any clarity.